### ISSUE TITLE
disabled Link color is not accessible

### ISSUE THREAD

**Issue by atikenny** [carbon-design-system/carbon#9114]
## disabled Link component is not accessible

The color of the Link component when it is disabled is not AA.

## Additional information

<img width="698" alt="Screenshot 2021-07-05 at 15 30 30" src="https://user-images.githubusercontent.com/6061509/124479024-1ade0680-dda6-11eb-83ba-fd0ab3fecf48.png">

**Comment by tw15egan** [comment]
Hey there! This actually seems to be a false positive from the a11y checker, as disabled items do not need to meet contrast requirements. Since they are inaccessible for keyboard/screen reader users, no information should be conveyed via a disabled item. 

https://www.w3.org/TR/WCAG21/#contrast-minimum

Specifically, 
>Incidental: Text or images of text that are part of an **inactive user interface component**

See also: 
https://github.com/carbon-design-system/carbon/issues/8741
https://github.com/carbon-design-system/carbon/issues/8273
